LM-1201/99-beta/cars.json

```diff
@@ -0,0 +1,19 @@
+{
+    "Sports Cars":[
+        {
+            "make": "Porsche",
+            "model": "911",
+            "stock": 9
+        },
+        {
+            "make": "Dodge",
+            "model": "Viper",
+            "stock": 4
+        },
+        {
+            "make": "Chevy",
```

Comment:
Spell out Chevrolet here

Reply:
One thing that Hank has said is he'd like more Cisco-typical JSON files. How about using even a simple host and port listing example? Such as filling in more of this:

```
{
    "servers":[
    "hostname": "nginx",
    "ip": "10.10.",
    "port": 3000,
    "inventory": 9,
    "ssh": {
        "publicKeys": [
          {
            "keyData": "..."
          }
        ]
    ]
}
```

Also note some two-word items that follow the Cisco API Style guidance for capitalization.